homeassistant/components/vacuum/__init__.py

```diff
@@ -217,8 +195,8 @@ def supported_features(self):
         raise NotImplementedError()
 
     @property
-    def status(self):
-        """Return the status of the vacuum cleaner."""
+    def state(self):
```

Comment:
You rewrite `state` property of `ToggleEntity`?

Reply:
You are right, it should no longer be a `ToggleEntity` as we are removing the toggle.